vmm/src/memory_manager.rs

```diff
@@ -309,15 +457,19 @@ impl MemoryManager {
                         / virtio_devices::VIRTIO_MEM_DEFAULT_BLOCK_SIZE
                         * virtio_devices::VIRTIO_MEM_DEFAULT_BLOCK_SIZE,
                 );
-                virtiomem_region = Some(MemoryManager::create_ram_region(
-                    &config.file,
-                    start_addr,
-                    size as usize,
-                    false,
-                    false,
-                    config.shared,
-                    config.hugepages,
-                )?);
+
+                if !use_custom_regions {
```

Comment:
Are we planning to support a `hotplug_method` and `hotplug_size` per configured memory zone? Or will both be global parameters that apply for all zones?

Reply:
No I think we talked about using `hotplug_method` and `hotplug_size` from `MemoryConfig` (or `--memory`) as global flags that will apply for all memory zones. Otherwise this would force us to define one more parameter to define global options for the memory zones, and I don't think we want that.